<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameClass</class>
 <widget class="QMainWindow" name="GameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pole0_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole0_1">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>70</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole0_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole1_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>145</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole1_1">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>145</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole1_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>145</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole2_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>215</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole2_1">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>215</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole2_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>215</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="countcheck">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Game.qrc"/>
 </resources>
 <connections/>
</ui>
